app/src/main/java/com/portfolio/tasky/entry/viewModels/LoginViewModel.kt

```diff
@@ -25,13 +25,13 @@ class LoginViewModel @Inject constructor(
 
 
 
-    fun emailChange(email : String){
+    fun emailChange(email : String) {
         mutableEmailChange.value = emailValidator.isValidEmailPattern(email)
     }
-    fun passwordChange(password : String){
+    fun passwordChange(password : String) {
         mutablePasswordChange.value = passwordPatternValidation.isPasswordPatternValid(password)
     }
-    fun areFieldsValid(){
+    fun areFieldsValid() {
```

Comment:
This function sounds like it returns a boolean which it doesn't. The fact that it updates a LiveData is considered a side-effect, so something you don't expect happens when reading the function name. 2 ways how you could improve that:
1. Rename the function to something like `validateFields()` or `checkIfFieldsValid()` which would make this more clear
2. You return `mutablePasswordChange.value == true && mutablePasswordChange.value == true` in this function and call it from somewhere else when you want to update the LiveData 

I personally prefer option 1.

Reply:
Okay

Done